src/accessibility-statement.md

```diff
@@ -16,10 +16,7 @@ AbilityNet has advice on [making your device easier to use](https://mcmw.ability
 
 ## How accessible this website is
 
-We know some parts of this website are not fully accessible:
-
-- some videos do not have a description and do not say they have no audio
-- sets of colours are missing clear groupings
+We are not aware of any accessibility issues on this website.
```

Comment:
@calvin-lau-sig7, there is actually no guidance in the sample statement about what to say in this section when a website is fully compliant. So I've more or less made this sentence up. (I've also googled a bit what other people say here, but found only bad or mediocre examples.)
Therefore, could you check if you are happy with this sentence. (All the other sentences I changed are directly from the sample statement.)

Reply:
I'm happy with this sentence.

There's probably connection between how often a review could be done and how often the accessibility statement is reviewed (1 year?), but we're not in a position to promise that we'll do that.

Reply:
Just a tiny style point, to make "we are" a contraction. Everything else looks good

```suggestion
We're not aware of any accessibility issues on this website.
```

Reply:
Thanks, I've just changed to 'We're'.

Regarding how often a statement should be reviewed, that is once a year. But the project will be out of our hands soon. There is probably a point to put this into the handover document. (I'll make a note of that to do later.)